Mathlib/Data/Set/Prod.lean

```diff
@@ -822,9 +813,19 @@ theorem dcomp_image_pi {f : ∀ i, α i → β i} (hf : ∀ i ∉ s, Surjective
   choose a hab hat using this
   exact ⟨a, hat, funext hab⟩
 
-theorem dcomp_image_univ_pi (f : ∀ i, α i → β i) (t : ∀ i, Set (α i)) :
-    (f _ ∘' ·) '' univ.pi t = univ.pi fun i ↦ f i '' t i :=
-  dcomp_image_pi (by simp) t
+@[deprecated (since := "2024-10-06")] alias dcomp_image_pi := piMap_image_pi
```

Comment:
Flagging this for other reviewers: you changed the statement of the lemma, renamed it and are adding an alias.
I think this is more helpful than harmful, but seems worth pointing out.

Reply:
No, I think it's just the GitHub view that is confusing.

Reply:
I changed the statement from `dcomp` to `Pi.map`. They are defeq but not reducibly defeq.

Reply:
Oh yeah ok I just misinterpreted that comment I think.